letsencrypt/client/init.sls

```diff
@@ -13,6 +13,7 @@ certbot_packages:
     - names: {{ client.source.pkgs }}
     - watch_in:
       - cmd: certbot_installed
+    - fromrepo: jessie-backports
```

Comment:
We should not control in formulas which repo is used when pkg installation is chosen. Instead, user should configure repositories adequately, for example using salt-formula-linux which also supports pinning repositories, etc.